core/tests/protractor/learnerFlow.js

```diff
@@ -298,9 +298,9 @@ describe('Learner dashboard functionality', function() {
     } else {
       // Create first exploration named 'Dummy Exploration'.
       await createDummyExplorationOnDesktop();
-      // Create a second exploration named 'Collection Exploration'.
+      // Create a second exploration named 'Collection Exp'.
       await workflow.createAndPublishExploration(
-        'Collection Exploration',
+        'Collection Exp',
```

Comment:
Why this name change? I think `Exploration` is easier to understand

Reply:
Agreed -- sorry, this was left over from a previous round of changes. Reverted!